cli/slsa-verifier/verify/verify_image.go

```diff
@@ -70,7 +72,11 @@ func (c *VerifyImageCommand) Exec(ctx context.Context, artifacts []string) (*uti
 		}
 	}
 
-	verifiedProvenance, outBuilderID, err := verifiers.VerifyImage(ctx, artifacts[0], provenance, provenanceOpts, builderOpts)
+	var verifiedProvenance []byte
```

Comment:
nit: we can remove defining these two variables, correct?

Reply:
done